tests/snapshots/list__list_mutants_in_all_trees_as_text.snap

```diff
@@ -379,6 +379,16 @@ src/numbers.rs:6:5: replace is_double -> bool with false
 src/numbers.rs:6:7: replace == with != in is_double
 src/numbers.rs:6:12: replace * with + in is_double
 src/numbers.rs:6:12: replace * with / in is_double
+src/numbers.rs:10:5: replace negate_int -> i32 with 0
+src/numbers.rs:10:5: replace negate_int -> i32 with 1
+src/numbers.rs:10:5: replace negate_int -> i32 with -1
+src/numbers.rs:10:5: replace - with  in negate_int
```

Comment:
I think the representation of mutants that just delete rather than replacing text needs to be specially handled. Maybe, if the replacement is "" we should change it to "delete {original} in {location}".

Reply:
That's a fair point, I'll try and make that change.